device.py

```
@@ -307,6 +307,39 @@ def write_usb_variables(output_file, device_properties: ConfigParser):
         output_file.write("CONFIG_TINYUSB_MSC_ENABLED=y\n")
         output_file.write("CONFIG_TINYUSB_MSC_MOUNT_PATH=\"/sdcard\"\n")
 
+def write_bluetooth_variables(output_file, device_properties: ConfigParser):
+    idf_target = get_property_or_exit(device_properties, "hardware", "target").lower()
+    has_bluetooth = get_boolean_property_or_false(device_properties, "hardware", "bluetooth")
+    if has_bluetooth:
+        output_file.write("# Bluetooth (NimBLE)\n")
+        output_file.write("CONFIG_BT_ENABLED=y\n")
+        output_file.write("CONFIG_BT_NIMBLE_ENABLED=y\n")
+        if idf_target == "esp32p4":
+            output_file.write(f"CONFIG_BT_NIMBLE_TRANSPORT_UART=n\n")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)